code/datums/supplypacks.dm

```diff
@@ -733,6 +786,45 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	containername = "plasmaman suit crate"
 	group = "Clothing"
 
+/datum/supply_packs/amogus
+	name = "Theatric Space Suits"
+	var/list/colored = list(
+		/obj/item/clothing/head/helmet/space,
+		/obj/item/clothing/suit/space,
+		/obj/item/clothing/head/helmet/space/syndicate,
+		/obj/item/clothing/suit/space/syndicate,
+		/obj/item/clothing/head/helmet/space/syndicate/green,
+		/obj/item/clothing/suit/space/syndicate/green,
+		/obj/item/clothing/head/helmet/space/syndicate/green/dark,
+		/obj/item/clothing/suit/space/syndicate/green/dark,
+		/obj/item/clothing/head/helmet/space/syndicate/orange,
+		/obj/item/clothing/suit/space/syndicate/orange,
+		/obj/item/clothing/head/helmet/space/syndicate/blue,
+		/obj/item/clothing/suit/space/syndicate/blue,
+	)
+	var/black = list(/obj/item/clothing/head/helmet/space/syndicate/black,
+					/obj/item/clothing/suit/space/syndicate/black,
+					/obj/item/clothing/head/helmet/space/syndicate/black/green,
+					/obj/item/clothing/suit/space/syndicate/black/green,
+					/obj/item/clothing/head/helmet/space/syndicate/black/blue,
+					/obj/item/clothing/suit/space/syndicate/black/blue,
+					/obj/item/clothing/head/helmet/space/syndicate/black/med,
+					/obj/item/clothing/suit/space/syndicate/black/med,
+					/obj/item/clothing/head/helmet/space/syndicate/black/orange,
+					/obj/item/clothing/suit/space/syndicate/black/orange,
+					/obj/item/clothing/head/helmet/space/syndicate/black/red,
+					/obj/item/clothing/suit/space/syndicate/black/red,
+					/obj/item/clothing/head/helmet/space/syndicate/black/engie,
+					/obj/item/clothing/suit/space/syndicate/black/engie)
+	cost = 300
+	containertype = /obj/structure/closet/crate/radiation
+	containername = "theatrical space suit crate"
+	group = "Clothing"
+	hidden = 1
+	
+/datum/supply_packs/amogus/New()
```

Comment:
you realize the git blame will forever associate this will your username, right?

Reply:
he does and does not care, he's on a different state of being than mere mortals

Reply:
Yea, and git blame will forever associate cult 4 with deity's name too, would I rather write /datum/supply_packs/theatric_space or /datum/supply_packs/amogus? Both mean the same thing but one of them is shorter

And don't say that amogus will lose its meaning, [https://bulbapedia.bulbagarden.net/wiki/Amoonguss_(Pok%C3%A9mon)](it never did)